lib/node_modules/@stdlib/stats/base/dnanmskrange/benchmark/benchmark.js

```diff
@@ -21,17 +21,38 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
+var uniform = require( '@stdlib/random/base/uniform' );
+var bernoulli = require( '@stdlib/random/base/bernoulli' );
+var uint8Array = require( '@stdlib/random/array/bernoulli' );
```

Comment:
Use the same approach as in the examples. Namely, just use the `factory` method from `base/bernoulli`. Applies here and to the other benchmarks.